bower.json

```diff
@@ -5,41 +5,41 @@
   "authors": [
     "The Polymer Authors"
   ],
-  "private": true,
-  "homepage": "https://github.com/PolymerElements/app-layout",
+  "license": "https://polymer.github.io/LICENSE.txt",
+  "homepage": "https://elements.polymer-project.org/elements/app-layout",
```

Comment:
@blasten you are right! I proposed for tedium: https://github.com/PolymerElements/app-layout/issues/293#issuecomment-237876465